docs/configuration.md

```diff
@@ -225,13 +231,23 @@ target_match:
   [ <labelname>: <labelvalue>, ... ]
 target_match_re:
   [ <labelname>: <regex>, ... ]
+  
+# Matchers that have to be fulfilled in the alerts to be muted 
+# with support of negative matching. 
+target_matchers:
+  [ <labelname>: <labelvalue>, <labelname>: <regex>, ... ]
 
 # Matchers for which one or more alerts have to exist for the
 # inhibition to take effect.
 source_match:
   [ <labelname>: <labelvalue>, ... ]
 source_match_re:
   [ <labelname>: <regex>, ... ]
+  
+# Matchers for which one or more alerts have to exist for the
+# inhibition to take effect with support of negative matching.
```

Comment:
As above: "A list of matchers for which one or more alerts have to exist for the inhibition to take effect."